Gravity/Gravity.Test.Integration/RSAPI_IntegrationTest.cs

```diff
@@ -319,10 +320,10 @@ public void Valid_Gravity_Object_Read_Field_Type<T>(string objectPropertyName, T
                 GravityLevelOne testObject = new GravityLevelOne();
                 testObject.Name = "TestObjectRead_" + objectPropertyName + Guid.NewGuid().ToString();
 
-                Guid testObjectTypeGuid = testObject.GetType().GetCustomAttribute<RelativityObjectAttribute>(false).ObjectTypeGuid;
-                Guid nameFieldGuid = BaseDto.GetCustomAttributeOfProperty<GravityLevelOne, Guid>("Name", x => x.FieldGuid);
-                Guid testFieldGuid = BaseDto.GetCustomAttributeOfProperty<GravityLevelOne, Guid>(objectPropertyName, x => x.FieldGuid);
-                RdoFieldType fieldType = (RdoFieldType)BaseDto.GetCustomAttributeOfProperty<GravityLevelOne, int>(objectPropertyName, x => x.FieldType);
+                Guid testObjectTypeGuid = testObject.GetCustomAttibutes<RelativityObjectAttribute>().ObjectTypeGuid;
+                Guid nameFieldGuid = testObject.GetCustomAttibute<RelativityObjectFieldAttribute>("Name").FieldGuid;
```

Comment:
The [`nameof` operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/nameof) would look nice here.

Reply:
(but I'll accept without it)

Reply:
Don't disagree with you here, maybe we can put into issue?

Reply:
I can clean it up post-merge.